.github/workflows/rector_ci.yaml

```diff
@@ -10,11 +10,31 @@ jobs:
     rector-ci:
         runs-on: ubuntu-latest
         steps:
-            -   uses: actions/checkout@v2
+            -
+                uses: actions/checkout@v2
+                with:
+                    ref: ${{ github.event.pull_request.head.ref }}
+                    token: ${{ secrets.ACCESS_TOKEN }}
+
             -
                 uses: shivammathur/setup-php@v1
                 with:
                     php-version: 7.3
                     coverage: none
+
             -   run: composer install --no-progress
             -   run: composer rector-ci
+
+            -
+                name: Check for modified files
+                id: git-check
+                run: echo ::set-output name=modified::$(if git diff --exit-code -s; then echo "false"; else echo "true"; fi)
+
+            -   name: Push changes
+                if: steps.git-check.outputs.modified == 'true'
+                run: |
+                    git config --global user.name 'rector-bot'
+                    git config --global user.email 'tomas@getrector.org'
+                    git commit -am "Bot commit"
```

Comment:
@TomasVotruba i would love to improve this commit message, please do you have any suggestions?